lib/std/compress/zlib.zig

```diff
@@ -170,3 +257,19 @@ test "sanity checks" {
         testReader(&[_]u8{ 0x78, 0xda, 0x03, 0x00, 0x00 }, ""),
     );
 }
+
+test "compress data" {
+    const allocator = testing.allocator;
+    const rfc1951_txt = @embedFile("rfc1951.txt");
```

Comment:
You'll need to rebase on master. This got moved to `testdata/rfc1951.txt` in https://github.com/ziglang/zig/commit/e6a4e87f69cdb8c76b24f136c945694db69f9b50